<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFSimplePlotWidget</class>
 <widget class="QWidget" name="QFSimplePlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Overlay Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLogX">
       <property name="text">
        <string>log X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkLogY">
       <property name="text">
        <string>log Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGrid">
       <property name="text">
        <string>grid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkKey">
       <property name="text">
        <string>show key</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFPlotter" name="plot" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>&amp;Help</string>
       </property>
       <property name="icon">
        <iconset resource="../lib/lib.qrc">
         <normaloff>:/lib/help/help.png</normaloff>:/lib/help/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFPlotter</class>
   <extends>QWidget</extends>
   <header>qfplotter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QFSimplePlotWidget</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkKey</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFSimplePlotWidget</receiver>
   <slot>updatePlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLogX</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFSimplePlotWidget</receiver>
   <slot>emitLogX(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLogY</sender>
   <signal>toggled(bool)</signal>
   <receiver>QFSimplePlotWidget</receiver>
   <slot>emitLogY(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showHelp()</slot>
  <slot>closeClicked()</slot>
  <slot>updatePlot()</slot>
  <slot>emitLogX(bool)</slot>
  <slot>emitLogY(bool)</slot>
 </slots>
</ui>
